### ISSUE TITLE
adding new metafield produces error

### ISSUE THREAD

**Issue by Eflores89** [charliebone/shopifyr#9]
Hi, I have been following the readme and get the following error when trying to add a metafield (it is same code as in the readme and I can see the new product added).

```
newMetafield <- shop$createMetafield("products", resourceId=newProduct$id, metafield=metafield)
Error in private$.request(slug = slug, reqType = reqType, data = data,  : 
  Error parsing response body : can't be blank; c("can't be blank", "is not included in the list")

```
Here is my session:

```
> sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shopifyr_0.6.4  httr_1.4.0      RCurl_1.95-4.12 bitops_1.0-6    inegiR_3.0.0   
[6] devtools_2.1.0  usethis_1.5.1  

loaded via a namespace (and not attached):
 [1] zoo_1.8-6         clisymbols_1.2.0  tidyselect_0.2.5  remotes_2.1.0    
 [5] purrr_0.3.2       lattice_0.20-38   testthat_2.1.1    XML_3.98-1.20    
 [9] rlang_0.4.0       pkgbuild_1.0.3    later_0.8.0       pillar_1.4.2     
[13] glue_1.3.1        tibbletime_0.1.2  withr_2.1.2       sessioninfo_1.1.1
[17] plyr_1.8.4        stringr_1.4.0     commonmark_1.7    memoise_1.1.0    
[21] callr_3.3.0       httpuv_1.5.1      ps_1.3.0          curl_3.3         
[25] Rcpp_1.0.1        promises_1.0.1    openssl_1.4       backports_1.1.4  
[29] desc_1.2.0        rversions_2.0.0   pkgload_1.0.2     jsonlite_1.6     
[33] fs_1.3.1          askpass_1.1       digest_0.6.20     stringi_1.4.3    
[37] processx_3.4.0    dplyr_0.8.3       RJSONIO_1.3-1.2   grid_3.6.1       
[41] rprojroot_1.3-2   cli_1.1.0         tools_3.6.1       magrittr_1.5     
[45] tibble_2.1.3      crayon_1.3.4      tidyr_0.8.3       pkgconfig_2.0.2  
[49] xml2_1.2.0        prettyunits_1.0.2 lubridate_1.7.4   assertthat_0.2.1 
[53] roxygen2_6.1.1    rstudioapi_0.10   R6_2.4.0          compiler_3.6.1   
[57] git2r_0.26.1     
```

**Comment by charliebone** [comment]
Hi, this has been fixed in 7982e077dc6f17ec228d48b02c7cd16eb594994b. 

Basically the example metafield object to be updated is missing the required value_type parameter. However, the returned error messages were very unhelpful. I have fixed both the example itself but also the response parser to hopefully better translate the returned error(s) from Shopify.

**Comment by Eflores89** [comment]
thanks!
